test/com/gfredericks/test/chuck/clojure_test_test.clj

```diff
@@ -18,6 +18,24 @@
       (swap! c inc)
       (is (> @c 0)))))
 
+(deftest exception-detection-test
```

Comment:
Ah, I thought I had accidentally deleted the test while I was making my change. Renamed the test in the correct place.